### PR TITLE
Updated document and error.js file regards issue #224

### PR DESCRIPTION
**Pull request by DeepshikhaNEC**
Updated document and error.js file regards issue sleepTime value error #224

### CODE REVIEW

File: docs/stepbystep.md [hunk 1]
```diff
@@ -278,6 +279,12 @@ curl -X POST -H "Content-Type: application/json" -H "Accept: application/json" -
 }' 'http://localhost:1026/v1/contextEntities/LivingRoomSensor/attributes/sleepTime'
 ```
 
+In case of MQTT to retrieve configuration parameters from the Context Broker, it is required that the device should be provisioned using 'MQTT' as transport key. By default it will be considered 'HTTP' as transport.
+
+The parameter will be given as follows:
+
+"transport": "MQTT"
```
Comment (fgalan) [inline docs/stepbystep.md:286]:
```suggestion
```
"transport": "MQTT" 
```
```

**Comment by fgalan** [comment]
Maybe the same document addition and fix should be done in IOTA-UL repository (https://github.com/telefonicaid/iotagent-ul/)? Both share a more or less common structure for the MQTT transport...

**Comment by fgalan** [comment]
Question:

> Updated document and error.js file regards issue sleepTime value error #224

I understand that although sleepTime is mentioned in #224, the PR at the end has nothing to do with sleepTime and that it is just a matter of improving documentation. Is my understanding correct?

File: docs/stepbystep.md (hunk 1 above)
Comment (DeepshikhaNEC) [reply docs/stepbystep.md]: Fixed in b7780b9

**Comment by DeepshikhaNEC** [comment]
@fgalan Hi, I have updated the file as per your suggestion.


> Question:
> 
> > Updated document and error.js file regards issue sleepTime value error #224
> 
> I understand that although sleepTime is mentioned in #224, the PR at the end has nothing to do with sleepTime and that it is just a matter of improving documentation. Is my understanding correct?

Yes your understanding is correct. Thanks!

**Comment by DeepshikhaNEC** [comment]
@fgalan Is CNR update required for this issue?

**Comment by fgalan** [comment]
> @fgalan Is CNR update required for this issue?

Don't think so. From the point of view of code changes, this PR only introduces a little adjust in a log trace.

File: docs/stepbystep.md [hunk 2]
```diff
@@ -278,6 +279,12 @@ curl -X POST -H "Content-Type: application/json" -H "Accept: application/json" -
 }' 'http://localhost:1026/v1/contextEntities/LivingRoomSensor/attributes/sleepTime'
 ```
 
+In case of MQTT to retrieve configuration parameters from the Context Broker, it is required that the device should be provisioned using 'MQTT' as transport key. By default it will be considered 'HTTP' as transport.
+
+The parameter will be given as follows:
+
+```"transport": "MQTT"```
+
```
Comment (fgalan) [inline docs/stepbystep.md:287]: Looking to PR https://github.com/telefonicaid/iotagent-ul/pull/433, maybe this fragment should be moved to https://github.com/telefonicaid/iotagent-json/blob/master/docs/usermanual.md#configuration-retrieval ?

**Pull request merged**